web/libs/ui/src/lib/enterprise-badge/enterprise-badge.module.scss

```diff
@@ -0,0 +1,35 @@
+.badge {
+  display: inline-block;
+  border-radius: 4px;
+  background: linear-gradient(135deg, #FFA663 0%, #FF7557 51.56%, #E37BD3 100%);
+  vertical-align: middle;
+  margin: 0 8px;
```

Comment:
Margins should not be defined by atomic components, I will delegate this responsibility to higher components or the scaffolding given that depending on where to place it there might be different spacing needs

Reply:
That makes sense. But right now I'm just moving the existing component to the library. Somehow we already had these styles in it https://github.com/HumanSignal/label-studio/pull/7173/files#diff-a8632dfbab038917426ca3becd12f72a8d553418e432ca6667c66eb66aff7651L6